Client/Frontend/Home/ActivityStreamTopSitesCell.swift

```diff
@@ -323,51 +323,67 @@ struct ASTopSiteSourceUX {
     static let CellIdentifier = "TopSiteItemCell"
 }
 
-protocol ASHorizontalScrollDelegate {
-    func numberOfVerticalItems(estimatedItems: Int) -> Int
-    func numberOfHorizontalItems(estimatedItems: Int) -> Int
+protocol ASHorizontalLayoutDelegate {
+    func numberOfVerticalItems() -> Int
+    func numberOfHorizontalItems() -> Int
 }
 
 /*
  This Delegate/DataSource is used to manage the ASHorizontalScrollCell's UICollectionView. 
  This is left generic enough for it to be re used for other parts of Activity Stream.
  */
 
-class ASHorizontalScrollSource: NSObject, UICollectionViewDelegate, UICollectionViewDataSource, ASHorizontalScrollDelegate {
+class ASHorizontalScrollCellManager: NSObject, UICollectionViewDelegate, UICollectionViewDataSource, ASHorizontalLayoutDelegate {
+
+    var _content = [TopSiteItem]()
+    var content: [TopSiteItem] {
```

Comment:
I've removed this. It has weird issues with items going missing when switching orientations. Now I just always get 12 items which makes sure that there are enough items for each orientation.